src/adr/0051-email-rate-limiting.md

```diff
@@ -0,0 +1,40 @@
+# Email rate limiting
+
+Date: 2021-01-10
+
+## Status
+
+// todo
+
+## Context
+
+Rate limiting is one of the most popular features of any service. They limit how many emails you can send/receive from/to
+each email account over a given period of time.  
```

Comment:
I would rather say it is a common feature. 

Give examples: SaaS is one. Give a link: https://www.fastmail.help/hc/en-us/articles/1500000277382-Account-limits#sending/receiving

Another exemple is rate limiting to specific remote host. When I was a student we had a rule on our mailing list servers of no more than 5 mail.sec to GMail not to be considered as Spammer, and traffic would be smoothed if needed. (we did set this up using Postfix)